src/components/Undo/Undo.scss

```diff
@@ -0,0 +1,69 @@
+.undo {
+  -moz-box-align: center;
+  -moz-box-pack: center;
+  display: -moz-box;
```

Comment:
Yeah, this is the old spec; you should definitely use the new (unprefixed) spec, this is a good guide here: https://css-tricks.com/snippets/css/a-guide-to-flexbox/